tools/settings.py

```diff
@@ -96,6 +96,10 @@
     'WASM_OBJECT_FILES',
     'WASM_WORKERS',
     'BULK_MEMORY',
+    'MIN_SAFARI_VERSION',
+    'MIN_CHROME_VERSION',
+    'MIN_FIREFOX_VERSION',
+    'MIN_NODE_VERSION',
```

Comment:
Hmm... this is kind of shame, but I guess its needed.

Reply:
Yeah agreed; but if we don't want to just follow the default at compile time and have consistent feature control at both compile and link time I don't really see another way.

Reply:
Actually, once these features are on by default and we are just lowering them away in Binaryen when disabled, we could pull this out again, as long as we don't care about older LLVM versions that might have different defaults.

Reply:
I actually think it might be clearer to get the lowering passes working first, otherwise we start to allow these settings at compile time and the try to dis-allow them later, once the lowering pass makes them irrelevant.

Reply:
That would be ideal; I think i have nontrapping-fp working and am partway through bulk-memory; we'd also need to be ready to ensure that all of our tests pass in that mode before they're ready, so it's good to have this in the back pocket if the change lands upstream before it's ready.